web/profiles/custom/yalesites_profile/modules/custom/ys_views_basic/src/Plugin/Field/FieldWidget/ViewsBasicDefaultWidget.php

```diff
@@ -179,6 +179,26 @@ public function formElement(
       ],
     ];
 
+    $form['group_user_selection']['markup_pre_limit'] = [
+      '#type' => 'markup',
+      '#markup' => '<div class="grouped-items">',
+    ];
+
+    $form['group_user_selection']['limit'] = [
+      '#title' => $this->t('Limit to'),
+      '#description' => $this->t('Enter 0 to show all items.'),
+      '#suffix' => $this->t('item(s)'),
+      '#type' => 'number',
+      '#default_value' => ($items[$delta]->params) ? $this->viewsBasicManager->getDefaultParamValue('limit', $items[$delta]->params) : 0,
+      '#min' => 0,
+      '#required' => TRUE,
+    ];
+
+    $form['group_user_selection']['markup_post_limit'] = [
+      '#type' => 'markup',
+      '#markup' => '</div>',
+    ];
+
```

Comment:
I never see this pattern in Drupal where HTML is started in one place and ended in another. (Talking about the wrapper div). Although it is extremely common in WordPress FWIW.

I suggest either using #prefix and #suffix on the limit item:
```
    $form['group_user_selection']['limit'] = [
      '#title' => $this->t('Limit to'),
      '#description' => $this->t('Enter 0 to show all items.'),
      '#prefix' => '<div class="grouped-items">',
      '#suffix' => $this->t('item(s)') . '</div>',
      '#type' => 'number',
      '#default_value' => ($items[$delta]->params) ? $this->viewsBasicManager->getDefaultParamValue('limit', $items[$delta]->params) : 0,
      '#min' => 0,
      '#required' => TRUE,
    ];
```
Or using a wrapping element such as a fieldset or a container:
```
    $form['group_user_selection']['limit_group'] = [
      '#type' => 'container',
      '#attributes' => [
        'class' => ['grouped-items'],
      ],
    ];
    $form['group_user_selection']['limit_group']['limit'] = [
      '#title' => $this->t('Limit to'),
      '#description' => $this->t('Enter 0 to show all items.'),
      '#suffix' => $this->t('item(s)'),
      '#type' => 'number',
      '#default_value' => ($items[$delta]->params) ? $this->viewsBasicManager->getDefaultParamValue('limit', $items[$delta]->params) : 0,
      '#min' => 0,
      '#required' => TRUE,
    ];
```

Reply:
Fixed. I actually removed all of that and essentially copied what Views does, so no suffix.